BigQuery/15_convert_schema_for_labeling.sql

```diff
@@ -0,0 +1,50 @@
+SELECT 
+subject_id as id,
+row_number() over (partition by subject_id, hadm_id, stay_id order by chart_hour asc) as ICULOS,
+admittime as HospAdmTime,
+0 as Unit1,
+1 as Unit2,
+age as Age,
+gender as Gender,
+platelets as Platelets,
+WBC,
+ptt as PTT,
+hemoglobin as Hgb,
+hematocrit as Hct,
+troponin_I as TroponinI,
+bilirubin_total as Bilirubin_total,
+potassium as Potassium,
+phosphate as Phosphate,
+magnesium as Magnesium,
+lactate as Lactate,
+glucose as Glucose,
+bilirubin_direct as Bilirubin_direct,
+creatinine as Creatinine,
+chloride as Chloride,
+calcium as Calcium,
+0 as Alkalinephos,
+0 as BUN,
+0 as AST,
+PaCO2,
+SaO2,
+PaO2,
+pH,
+FiO2,
+bicarbonate as HCO3,
+base_excess as BaseExcess,
+EtCO2 as EtCO2,
+resp_rate as Resp,
+diastolic_bp as DBP,
+mean_arterial_bp as MAP,
+systolic_bp as SBP,
+temperature as Temp,
+spo2 as O2Sat,
+heart_rate as HR,
+ventilated as vent_status,
+blood_cx as blood_cx_order,
+urine_cx as urine_cx_order,
+sputum_cx as sputum_cx_order,
+antibiotics_amount as iv_abx_dose,
+abx_orders as iv_abx_ordered
+FROM `elevated-pod-307118.physionet.final_schema`
+WHERE total_los>8
```

Comment:
This is only going to select folks with at least 9 hours of time in the ICU. (It will change if/when this gets higher resolution data.) This may be ok if it's specific to the labeler, but it may need to return everything unless we're just discounting the first 8.99+ hours of time in the ICU.

Nit: Add spaces around the `>` operator.